### DISCUSSION TITLE
Gopro 7 Linear Mode Not working by QR code

### DISCUSSION THREAD

**Discussion by jamalionline333** [gopro/labs#621] [Feature Requests]
Why QR Code is not working to make Video in Linear Mode for Gopro7 black? I need LInear only because my screen is not working and its a blessing to set camera without touch screen

**Comment by dnewman-gpsw** [comment]
I'm away for a few days, I would need to look a HERO7 code to see if Linear was supported via another method/command.  Ping me Thursday, if you haven't heard. As there will be no newer Labs releases on HERO7, you can always do post Wide to Linear conversion, which I believe can offer more control and quality than in-camera linear.

**Reply by dnewman-gpsw** [reply]
Linear is working via QR code. The command is fL. 
The issue is https://gopro.github.io/labs/control/custom/  is using fL0 with is a command for FOV and Zoom, with is not supported on HERO7.
In the mobile App, it uses on fL.
